test/unit/transform/additionalfeeds.py

```diff
@@ -29,6 +29,34 @@ def test_transform_weekly_depth_charts(self):
         # Assert
         self.assertIsInstance(result, dict)
 
+    def test_transform_daily_change_log(self):
+        # Act
+        result = self.gft.transform_daily_change_log()
+
+        # Assert
+        self.assertIsInstance(result, dict)
+
+    def test_transform_daily_transactions(self):
+        # Act
+        result = self.gft.transform_daily_transactions()
+
+        # Assert
+        self.assertIsInstance(result, dict)
+
+    def test_transform_seasons(self):
+        # Act
+        result = self.gft.transform_seasons()
+
+        # Assert
+        self.assertIsInstance(result, dict)
+
+    def test_transform_weekly_injuries(self):
+        # Act
+        result = self.gft.transform_weekly_injuries
```

Comment:
without parenthesis at the end, it would not pass the test.

Reply:
Added